tcms/testplans/tests/test_views.py

```diff
@@ -54,3 +56,167 @@ def test_view_tags_without_permissions(self):
         # assert tag actions are shown
         self.assertNotContains(response, 'Add Tag')
         self.assertContains(response, '<span class="disabled grey">%s</span>' % _('Remove'))
+
+
+class NewPlanViewTest(BasePlanTest):
```

Comment:
This is unrelated so it was better to leave it as it was. If you revert it that would be better.